bot/exts/evergreen/xkcd.py

```diff
@@ -69,6 +69,8 @@ async def fetch_xkcd_comics(self, ctx: Context, comic: Optional[str]) -> None:
                     return
 
         embed.title = f"XKCD comic #{info['num']}"
+        embed.description = info['alt']
+        embed.url = f"{BASE_URL}/{comic}"
```

Comment:
This won't work if the user does `.xkcd latest` as the `comic` var isn't updated to be the comic number.

Reply:
ok

Reply:
Fixed.